assets/styles.css

```diff
@@ -0,0 +1,202 @@
+/* Apple Clone Website Styles */
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap");
+
+/* CSS RESET */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  align-items: center;
+}
+
+h3 {
+  margin: 20px auto;
+}
+
+.container {
+  width: 100%;
+  max-width: 1440px;
+  margin: auto;
+  font-family: "Lato", sans-serif;
+}
+
+.flex {
+  display: flex;
+}
+
+.color-white {
+  color: #fff;
+}
+
+.hero {
+  min-height: 77vh;
+  background-image: url("./images/your_verse_hero.jpg");
+  background-repeat: no-repeat;
+  background-position: 50% 100%;
+  background-size: 1440px 1315px;
+}
+
+.hero-main {
+  text-align: center;
+  padding-top: 40px;
+}
+
+.nav-items {
+  width: 900px;
+  margin: 0 auto;
+  align-items: center;
+  justify-content: center;
+  border-radius: 12px;
+  padding: 0 10px;
+  position: relative;
+  text-align: center;
```

Comment:
- [ ] You use `text-align: center;` 4 times in your css. For good coding practices, and so that you don't repeat yourselves, you can use multiple classes for any element. So for example,

  If you use this lines in many classes:
  ```
  color: #fff;
  background-color: #000;
  ```
   and you use it for .class1, .class2, and .class3 
  you can put them inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
    background-color: #000
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  <!-- this will have font size of 24px, color of #fff and background-color of #000 -->
  ...
  <div class="class2 common-class">  <!-- this will have font size of 30px, color of #fff and background-color of #000 -->
  ...
  <footer class="class3 common-class">   <!-- this will have font size of 10px, color of #fff and background-color of #000 -->
  ...
  ```